DEVELOPMENT.md

```diff
@@ -0,0 +1,54 @@
+[![Build Status](https://travis-ci.org/weldr/bdcs.svg?branch=master)](https://travis-ci.org/weldr/bdcs)
+[![Coverage Status](https://coveralls.io/repos/github/weldr/bdcs/badge.svg?branch=master)](https://coveralls.io/github/weldr/bdcs?branch=master)
+
+
+Preparing local development environment for Haskell
+===================================================
+
+For development we use the latest upstream versions:
+
+1) Remove the standard `haskell-platform` and `ghc-*` RPMs if you have them installed
+2) Download version **8.0.2** of the generic Haskell Platform distribution from
+   https://www.haskell.org/platform/linux.html#linux-generic
+3)
+```
+$ tar -xzvf haskell-platform-8.0.2-unknown-posix--minimal-x86_64.tar.gz
+$ sudo ./install-haskell-platform.sh
+```
+4) Add `/usr/local/bin` to your PATH if not already there!
+5) Install build dependencies:
+```
+# dnf -y install xz-devel zlib-devel glib2-devel gobject-introspection-devel ostree-devel
+```
+
+**NOTE:** On RHEL 7 `ostree-devel` is part of the Atomic Host product!
+
+
+Building the project locally
+============================
+
+`cabal` is used to install and manage Haskell dependencies from upstream.
+
+```
+$ cd src/ && cabal sandbox init && cabal install
+```
+
+Executing unit tests
+====================
+
+    $ cabal sandbox init
+    $ cabal install --dependencies-only --enable-tests
+    $ cabal test
```

Comment:
Have you confirmed these in a clean install? I keep having problems with hlint and happy when I do this inside docker (I think I fixed it last time by installing the fedora version of happy).

Reply:
I have not - this file was just a straight copy over from what was in README.md before without modification.  However, I have not had problems with this in the past.  I'd be interested in seeing what you're seeing.